Cartography.xcodeproj/project.pbxproj

```diff
@@ -788,6 +788,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				APPLICATION_EXTENSION_API_ONLY = YES;
+				CODE_SIGN_IDENTITY = "Mac Developer";
```

Comment:
But this changes it from doing no codesigning to signing with the _Mac Developer_ identity, right?

Btw, if you're only dealing with iOS, you can use `carthage build --platform ios` to skip building for Mac altogether.